tools/emscripten.py

```diff
@@ -979,16 +981,17 @@ def create_receiving(function_exports, tag_exports, library_symbols):
 
   receiving.append('\nfunction assignWasmExports(wasmExports) {')
   for sym, sig in exports.items():
+    is_function = sig is not None
```

Comment:
I'm not totally sure how Emscripten is currently handling tags, but at least in wasm, tags have signatures too (the return value is void and the arguments are the payload type). In C++ we only use i32-payload tags, and maybe that's fine for Emscripten, but if we want this linking method to be fully general it should handle other tag types.

Reply:
(See the `= None` assignment above)